### PR TITLE
Allow to search users by member_uid

### LINKED ISSUE #343: Users want to search others by member_uid
Before angular removal #272 users could search in users list, timeoverflow.org/members, by member id (número de socio)

We want this feature back :pray:

### PR DESCRIPTION
**Pull request by markets**
Closes #343 

> Users want to search others by member_uid

Implemented using current search based on `Ransack`. I followed [this guide](https://github.com/activerecord-hackery/ransack/wiki/Using-Ransackers) to map the integer to a char and make it work with the pg ILIKE.

**EXTRA** Allow to sort by the `member_uid` column

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -83,7 +83,7 @@
     ]
   end
 
-  config.include Devise::TestHelpers, type: :controller
+  config.include Devise::Test::ControllerHelpers, type: :controller
```
Comment (markets) [inline spec/spec_helper.rb]: This raises a warning. Originally pushed in https://github.com/coopdevs/timeoverflow/commit/75aa85d33524f2d9a55a27be9ee9e105b3ff93f1, and overridden by #333.

**Comment by markets** [comment]
Current build is "unstable" due to other tests (not related to these changes), as noted by @rewritten in #351: https://github.com/coopdevs/timeoverflow/pull/351/commits/cd259683aebc35b92aa712c7e652c2d464486fda.

**Comment by sauloperez** [comment]
Take a look at @rewritten 's PR cause it looks like he's working on it.

**Comment by markets** [comment]
👍 seems that it was solved there, I'll rebase on top when it get merged to develop, because both branches are also touching the ransack form.

**Comment by markets** [comment]
Updated from develop, but 1 test is failing (maybe the ransacker ¯\_(ツ)_/¯ ??), I'll take a closer look and I'll ping you for review.

**Comment by markets** [comment]
@enricostano @sauloperez Finally CI is green ✅ 😃

**Comment by markets** [comment]
Updated with latest `develop` and fixed conflitcts in `app/views/users/index.html.erb` file.

**Comment by sseerrggii** [comment]
I changed the label to "need review" because you didn't merge it and I can not test it

**Comment by enricostano** [comment]
Deployed on https://staging.timeoverflow.org/

**Comment by enricostano** [comment]
Tested with @sseerrggii :green_apple:

**Pull request merged**